examples/quantization_w8a8_fp8/qwen3_next_example.py

```diff
@@ -0,0 +1,47 @@
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from llmcompressor import oneshot
+from llmcompressor.modifiers.quantization import QuantizationModifier
+from llmcompressor.utils import dispatch_for_generation
+
+MODEL_ID = "Qwen/Qwen3-Next-80B-A3B-Instruct"
+
+# Load model.
+model = AutoModelForCausalLM.from_pretrained(
+    MODEL_ID,
+    torch_dtype="auto",
+    device_map="auto",  # Use device_map for large model
```

Comment:
you dont need to! we call dispatch_for_generation in the data free pipeline which will map correctly